features/step_definitions/operators.rb

```diff
@@ -26,6 +26,31 @@
   end
 end
 
+Given /^I wait(?: up to #{NUMBER} minutes)? for all clusteroperators to be ready$/ do |timeout_in_minutes|
+  ensure_admin_tagged
+  timeout_in_minutes ||= 30
```

Comment:
Are we going to wait 30 minutes for all cluster operators to be ready? This looks very long.

Reply:
@jhou1 each co is checked with a timeout of 30 minutes for each.  In line #40, it will break out of the loop upon the first failed check.  So I think we are ok.

Reply:
@jhou1 what timeout value do you have in mind?

Reply:
I assume on a healthy cluster we would need less than 30 seconds.

Reply:
btw, do we need to take into account of the other statues from a co, like `available: true`, `degraded: false` ?

Reply:
@jhou1 it depends on how the step is used.  If we keep it as is, it can be used to poll COs that takes longer to come up. 
 @weliang1 WDYT?  Currently, @weliang1 just wants to check when the COs are `ready`.

Reply:
@jhou1 BTW, the value for upgrade check is [20 minutes](https://gitlab.cee.redhat.com/aosqe/jenkins-jcasc-n/-/blob/master/job-dsl-lib/UpgradeCIUtils.groovy#L549)

Reply:
@jhou1 @weliang1's golang example if only 30 seconds.   But I think he used it after the cluster has finished spun up/upgraded.   So it does depends on when the step is called.

Reply:
> @jhou1 it depends on how the step is used. If we keep it as is, it can be used to poll COs that takes longer to come up. @weliang1 WDYT? Currently, @weliang1 just wants to check when the COs are `ready`.

@pruan-rht For my test case, 30 seconds will be good enough.